wiring/inc/spark_wiring_cloud.h

```diff
@@ -41,9 +41,6 @@
 #include <functional>
 #include <type_traits>
 
-#define PARTICLE_DEPRECATED_API_DEFAULT_PUBLISH_SCOPE \
-        PARTICLE_DEPRECATED_API("Beginning with 0.8.0 release, Particle.publish() will require event scope to be specified explicitly.");
-
 #define PARTICLE_DEPRECATED_API_DEFAULT_SUBSCRIBE_SCOPE \
```

Comment:
This should also probably be removed, as I think there is no scope with subscribes either now.

@monkbroc

Reply:
I think it's fine to remove this deprecation message anyway since it's no longer relevant, i.e. we won't be requiring to specify one of the existing scopes explicitly (`MY_DEVICES`, `ALL_DEVICES`). As for the public event stream, @elijthomas could you please clarify if we finally disabled it?

Reply:
According to Eli, the public stream is disabled already.

Reply:
Thank you for checking, Sergey. In that case, I believe we can merge this PR.

Reply:
Sounds good. I've clarified this with Eli just now, and it basically doesn't matter whether you're using `MY_DEVICES` or `ALL_DEVICES` with your subscriptions and, accordingly, `PUBLIC` or `PRIVATE` with published events. Let's keep in mind that we also need to update the reference docs accordingly. cc @rickkas7

Reply:
Got it. thanks!...